src/main.tsx

```diff
@@ -13,6 +14,7 @@ const root = createRoot(container as HTMLElement)
 root.render(
   <React.StrictMode>
     <ReactQueryProvider>
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
not 100% sure i've instantiated this in the best place, but it works! and it looks similar to what's mentioned here in the docs:
https://tanstack.com/query/v4/docs/react/devtools#floating-mode

Reply:
Looks good to me.